modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -382,20 +393,42 @@ export default class Tileset3D {
   /**
    * Update visible tiles relying on a list of viewports
    * @param viewports - list of viewports
+   * @deprecated
    */
+  update(viewports: any[] | null = null) {
+    if (!viewports && this.lastUpdatedVieports) {
+      viewports = this.lastUpdatedVieports;
+    } else {
+      this.lastUpdatedVieports = viewports;
+    }
+    this.doUpdate(viewports);
+  }
+
+  async selectTiles(viewports: any[] | null = null): Promise<number> {
```

Comment:
JSDoc headers on all functions please, and avoid `any` if at all possible

Reply:
We don't use types from @deck.gl here because it will be circular reference.